samples/application-conf/file.properties

```diff
@@ -15,3 +15,7 @@ dbb.scriptMapping = ZunitConfig.groovy :: **/*.bzucfg
 #
 # Scanner mappings for application programs that require a custom scanner
 dbb.scannerMapping = ZUnitConfigScanner :: **/*.bzucfg
+
+#
+# file mapping for generated zUnit Test case programs (Cobol) to use a seperate set of libraries
+# cobol_testcase = true :: **/testcase/*.cbl
```

Comment:
In case the value `cobol_testcase ` is not defined in `file.properties` what about the following code in `Cobol.groovy` ?
```
	// Check if this a testcase
	isZUnitTestCase = (props.getFileProperty('cobol_testcase', buildFile).equals('true')) ? true : false
```

if `props.getFileProperty('cobol_testcase', buildFile)` return null you will have an NullPointerException exception.

Reply:
Apparently, I don't receive a NPE in the case of not providing a cobol_testcase file property.